tests/license/test_discovery.py

```diff
@@ -25,91 +27,60 @@ def license_pytest_path(data_dir) -> str:
-                    "scheme": "Trove",
-                },
-            ],
-            "links": [
-                {
-                    "note": "tl;dr legal",
-                    "url": "https://tldrlegal.com/license/mit-license",
-                },
-                {
-                    "note": "Wikipedia page",
-                    "url": "https://en.wikipedia.org/wiki/MIT_License",
-                },
-                {"note": "OSI Page", "url": "https://opensource.org/licenses/mit"},
-            ],
-            "name": "MIT/Expat License",
-            "other_names": [
-                {
-                    "name": "MIT",
-                    "note": "Because MIT has used many licenses for software, "
-                    "the Free Software Foundation considers MIT License"
-                    " ambiguous. The MIT License published on the OSI"
-                    " site is the same as the Expat License.",
-                },
-                {
-                    "name": "Expat",
-                    "note": "Because MIT has used many licenses for software,"
-                    " the Free Software Foundation considers MIT License"
-                    " ambiguous. The MIT License published on the OSI site"
-                    " is the same as the Expat License.",
-                },
-            ],
-            "superseded_by": None,
-            "keywords": ["osi-approved", "popular", "permissive"],
-            "text": [
-                {
-                    "media_type": "text/html",
-                    "title": "HTML",
-                    "url": "https://opensource.org/licenses/mit",
-                }
-            ],
+            "reference": "./MIT.html",
+            "isDeprecatedLicenseId": False,
+            "isFsfLibre": True,
+            "detailsUrl": "http://spdx.org/licenses/MIT.json",
+            "referenceNumber": "256",
+            "name": "MIT License",
+            "licenseId": "MIT",
+            "seeAlso": ["https://opensource.org/licenses/MIT"],
+            "isOsiApproved": True,
         }
     ]
 
 
 def test_match_license():
-    assert match_license("MIT License")["id"] == "MIT"
-    assert match_license("Expat")["id"] == "MIT"
+    assert match_license("MIT License")["licenseId"] == "MIT"
+    assert match_license("Expat")["licenseId"] == "MIT"
 
 
-def test_get_all_licenses_from_opensource():
-    assert len(get_all_licenses_from_opensource()) >= 88
-    assert get_all_licenses_from_opensource()[0]["id"]
+def test_get_all_licenses_from_spdx():
+    assert len(get_all_licenses_from_spdx()) > 300
+
+
+def test_get_opensource_license_data():
+    assert len(get_opensource_license_data()) >= 50
 
 
 def test_short_license_id():
     assert get_short_license_id("MIT License") == "MIT"
     assert get_short_license_id("Expat") == "MIT"
-    assert get_short_license_id("GPL 2.0") == "GPL-2.0"
+    assert get_short_license_id("GPL 2.0") == "GPL-2.0-only"
```

Comment:
`GPL-2.0-only` software are very rare. `GPL-2.0-or-later` is the most popular.

Reply:
Fixing with #108 
Thanks for warning me about that!